base/stats.go

```diff
@@ -500,6 +500,8 @@ type DatabaseStats struct {
 	SyncFunctionExceptionCount *SgwIntStat `json:"sync_function_exception_count"`
 	// The total number of times a replication connection is rejected due ot it being over the threshold
 	NumReplicationsRejectedLimit *SgwIntStat `json:"num_replications_rejected_limit"`
+	// The number incoming of concurrent replication connections
+	NumConcurrentReplications *SgwIntStat `json:"num_concurrent_replications"`
```

Comment:
Is there a reason to not name this to match the server context stat `num_replications_active` ?

Reply:
Can you clarify in the comments why these two are different, probably in both places?